tests/Local.postman_environment.json

```diff
@@ -109,9 +109,33 @@
 			"value": "",
 			"type": "default",
 			"enabled": true
+		},
```

Comment:
One of these days can you show me how the postman tests work?

Reply:
I'll add something to the reader